src/components/users/concerts/ConcertList.js

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import ConcertDetail from "./ConcertDetail";
+
+function ConcertList(props) {
+  const { concerts } = props;
+
+  return (
+    <>
+      {concerts.map((concert) => {
+        return <ConcertDetail concert={concert} />;
+      })}
```

Comment:
Hier fehlt noch die id

```suggestion
      {concerts.map((concert, i) => {
        return <ConcertDetail key={i} concert={concert} />;
      })}
```

Reply:
done